exercises/concept/go/.docs/introduction.md

```diff
@@ -0,0 +1,113 @@
+# Introduction
```

Comment:
It looks like the contents of this file match the concept's `about.md`, but it should be much closer to the `introduction.md`. As per [the docs](https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-introduction-md), the exercise's `introduction.md` file should give the student just enough context to figure out the solution themselves.

Reply:
Good idea, I've done this. The interplay of these 3 files is a source of confusion for me, it feels like it might be possible to find a better way to structure the information, although I realise there are reasons ...

Reply:
There are indeed reasons :) We do offer the option of using `configlet generate`: https://exercism.org/docs/building/configlet/generating-documents This allows you to basically copy the contents of the concept introduction to the exercise introduction without repeating it.